packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set dropdownDefault.twig__1.html

```diff
@@ -0,0 +1,6 @@
+<html><body>
+<div class="Dropdown">    Content
```

Comment:
Please minify content code as much as possible. Try trim text/content, remove redundant spaces.

Reply:
I think this is not an issue but a feature because the template is written like
```
<Dropdown>
....Content // 4 spaces before content
</Dropdown>
```
Writing it as `<Dropdown>Content</Dropdown>` will solve this. But as I wrote, this is not an issue.

Reply:
OK, it's not an issue to solve in this issue scope, BUT...

I think opposite. 

As far as I know every HTML minifier is removing all spaces between HTML tags and between starting tag, its content and its closing tag. 

If component (Dropdown in our case) would not be flex but any inline/inline-block element the single space will be rendered anyway and it is something that is unwanted and redundant. 

Single spaces at any count browser interprets as one (in our case one before "Content" word and one after.)

I think this behavior is wrong. I'd rather call for a new issue for this and discuss this topic further.

Reply:
We are aware of this behavior. Could you please file an issue?